test/Microsoft.AspNet.Mvc.ViewFeatures.Test/Rendering/HtmlHelperPasswordTest.cs

```diff
@@ -345,6 +345,40 @@ public static TheoryData PasswordFor_WithComplexExpressionsData
             Assert.Equal(expected, HtmlContentUtilities.HtmlContentToString(result));
         }
 
+        [Fact]
+        public void PasswordHelpers_UsesSpecifiedExpression()
+        {
+            // Arrange
+            var helper = DefaultTemplatesUtilities.GetHtmlHelper();
+
+            // Act
+            var passwordResult = helper.Password("Property1");
+            var passwordForResult = helper.PasswordFor(m => m.Property1);
```

Comment:
`[Theory]` or two `[Fact]`s. (`[Theory]` would be :ok: here because expressions work out to the same thing and output is identical.)